Source/Serilog.Exceptions/Destructurers/ReflectionBasedDestructurer.cs

```diff
@@ -109,6 +109,8 @@ public ReflectionBasedDestructurer(int destructuringDepth)
             return refId;
         }
 
+        private static object DestructureQueryable(IQueryable value) => value.Expression.ToString();

```

Comment:
I'm not sure we should be calling `ToString`. In another example below when we access a property that throws we add:

```
$"threw {innerException.GetType().FullName}: {innerException.Message}"
```

Perhaps we should hard code a constant string here. Thoughts @krajek?

Reply:
Yes, I would go the constant string way. If we want to eliminate evaluation of the property we should go all the way and just inform the user that `IQueryable skipped` or `IQueryable is not destructured`. Additionally, we would need to add a section to README with this PR to make sure clients of the library understand why we did that and that they can use custom destructurers if they want to do otherwise.

Reply:
I'd go with either "Skipped" or "IQueryable Skipped".

Reply:
Regarding the README additions, I'm unsure which section to add it in, to I just modified the warning section under Serilog.Exceptions.EntityFrameworkCore.